test/Controller/InsideElifeArticleControllerTest.php

```diff
@@ -46,6 +46,13 @@ public function it_has_metadata()
         $this->assertSame('Blog article impact statement', $crawler->filter('meta[name="description"]')->attr('content'));
         $this->assertSame('article', $crawler->filter('meta[property="og:type"]')->attr('content'));
         $this->assertSame('summary', $crawler->filter('meta[name="twitter:card"]')->attr('content'));
+        $this->assertEmpty($crawler->filter('meta[property="og:image"]'));
+        $this->assertSame('blog-article/1', $crawler->filter('meta[name="dc.identifier"]')->attr('content'));
```

Comment:
I wonder what will happen to existing annotations added during this private beta period, will they still be matched through the URL, or will they disappear and require to be ported to the new identifiers?

Reply:
They'll need porting, @nlisgo is on it.